Plugins/GitFlow/GitFlowForm.Designer.cs

```diff
@@ -459,11 +462,22 @@ private void InitializeComponent()
             this.lblRunCommand.TabIndex = 1;
             this.lblRunCommand.Text = "-";
             // 
+            // cbApplyToSubmodules
+            // 
+            this.cbApplyToSubmodules.AutoSize = true;
```

Comment:
Should this be in the Start branch group, would it not make sense to have this in next group?

Reply:
I don't think it should be inside the Start branch group since the checkbox applies for ALL commands. So it has to be above all steps to follow in my opinion.